src/ui-shell/sidenav/sidenav-menu.component.ts

```diff
@@ -53,17 +57,40 @@ export class SideNavMenu implements AfterContentInit {
 	 * Controls the visibility of the child `SideNavItem`s
 	 */
 	@Input() expanded = false;
+	/**
+	 * Controls the active status indicator on the menu if there is an active
+	 * child sidenav item.
+	 */
+	@Input() hasActiveChild = false;
 
 	@ContentChildren(SideNavItem) sidenavItems: QueryList<SideNavItem>;
 
+	protected findActiveChildren() {
+		if (this.sidenavItems.some(item => item.active)) {
+			this.hasActiveChild = true;
+		} else {
+			this.hasActiveChild = false;
+		}
+	}
+
+	protected activeItemsSubscription: Subscription;
+
 	ngAfterContentInit() {
 		setTimeout(() => {
 			this.sidenavItems.forEach(item => {
 				item.isSubMenu = true;
+				this.findActiveChildren();
+				this.activeItemsSubscription = item.toggled.subscribe(() => {
```

Comment:
This doesn't do quite what you intend - this won't track _all_ `selected` subscriptions just the most recent last child ... You'll want to make `activeItemsSubscription` into a `Subscription` instance and `.add` each subscription to that ... You may also need to handle updates to the sidenav items ... I believe `QueryList` has a change Observable available